docs/contribute/api-docs-contribution/workflows.md

```diff
@@ -0,0 +1,258 @@
+        
+    4. Get assistance from someone with admin authority in Bump.sh (reach out on #next-api-reference Slack channel) to deploy the new output file as described in [https://docs.bump.sh/help/publish-documentation/deploy-and-release-management/](https://docs.bump.sh/help/publish-documentation/deploy-and-release-management/) and [Bump.sh API admin instructions](https://docs.google.com/document/d/1xP_n2AusBhJcLmmJeCZZkAztzeykvkZtJ6la2XHz7Aw/edit?tab=t.0) (Elastic employees only). 
+        
+        :::{important}
+        You must ensure that you upload the file to the appropriate version in Bump.sh ("main", "v9", or "v8") as described in [https://docs.bump.sh/help/publish-documentation/branching/#deploy-to-a-specific-branch](https://docs.bump.sh/help/publish-documentation/branching/#deploy-to-a-specific-branch)
+
+        If the file that you upload is unchanged from whatever was last automatically deployed, you will receive a message that there's nothing new to upload and should just release the latest existing file (for the appropriate branch).
+        :::
+        
+            
+
+### Logstash 
+
+To publish content to [https://www.elastic.co/docs/api/doc/logstash](https://www.elastic.co/docs/api/doc/logstash) : 
+
+1. Check out the appropriate branch in the [logstash repo](https://github.com/elastic/logstash/tree/main) (e.g. "main" ).
+    
+2. Find the OpenAPI document in the `docs/static/spec/openapi` folder. For example: [https://github.com/elastic/logstash/blob/main/docs/static/spec/openapi/logstash-api.yaml](https://github.com/elastic/logstash/blob/main/docs/static/spec/openapi/logstash-api.yaml)
+    
+3. Optionally, lint and preview the document. For example, install [redocly](https://redocly.com/docs/cli/installation) and [bump.sh](https://docs.bump.sh/help/continuous-integration/cli/) CLI, then run:
+    
+    1. `redocly lint logstash-api.yaml --config redocly.yaml`
+        
+    2. `bump preview logstash-api.yaml`
+        
+4. Get assistance from someone with admin authority in Bump.sh (reach out on #next-api-reference Slack channel) to deploy the new output file as described in [https://docs.bump.sh/help/publish-documentation/deploy-and-release-management/](https://docs.bump.sh/help/publish-documentation/deploy-and-release-management/) and [Bump.sh API admin instructions](https://docs.google.com/document/d/1xP_n2AusBhJcLmmJeCZZkAztzeykvkZtJ6la2XHz7Aw/edit?tab=t.0) (Elastic employees only). 
+    
+    - If the file that you upload is unchanged from the previous one, you will receive a message that there's nothing new to deploy.  
+        
+
+### Observability Intake (serverless)
+
+To publish content to [https://www.elastic.co/docs/api/doc/observability-serverless](https://www.elastic.co/docs/api/doc/observability-serverless)
```

Comment:
IMO it's better to point to the individual process docs in the appropriate repos, in case they change.

```suggestion
To generate [https://www.elastic.co/docs/api/doc/observability-serverless](https://www.elastic.co/docs/api/doc/observability-serverless), refer to the [README](https://github.com/elastic/apm-managed-service/blob/main/docs/spec/openapi/README.md) 
```